tests/integration/helpers/cluster.py

```diff
@@ -624,7 +647,7 @@ def start(self, destroy_dirs=True):
 
             self.docker_client = docker.from_env(version=self.docker_api_version)
 
-            common_opts = ['up', '-d', '--force-recreate']
+            common_opts = ['up', '-d']
```

Comment:
Why removing force recreate? I guess it can have a side effects - if test1 start containers and will not remove them, 
test2 can get 'dirty' containers from 1st test run.

Reply:
It should be ok inside one integration test file. We just need to add separate names for different cases.

Reply:
Then same problem can happen if the SAME test will be fired twice.

Reply:
There is `docker-compose down` at the beginning of `start` and it will stop and remove all containers or fail, so `--force-recreate` is not required anymore